src/stru_multi/4C_stru_multi_microstatic.cpp

```diff
@@ -717,10 +730,44 @@ void MultiScale::MicroStatic::output(
   }
 }  // MultiScale::MicroStatic::output()
 
+/*----------------------------------------------------------------------*
+ |  write runtime output (public)
+ *----------------------------------------------------------------------*/
+void MultiScale::MicroStatic::runtime_output(
+    const std::pair<double, int>& output_time_and_step, const std::string& section_name) const
+{
+  const auto visualization_manager_ptr_ = micro_vtu_writer_ptr_->visualization_manager_ptr();
+  auto& visualization_data = visualization_manager_ptr_->get_visualization_data();
+
+  std::vector<double>& out_cmat =
+      visualization_data.get_field_data<double>("tangent_stiffness_tensor_cmat");
+  out_cmat.clear();
+
+  for (int i = 0; i < 6; i++)
+  {
+    for (int j = 0; j < 6; j++)
+    {
+      const auto a = macro_cmat_(i, j);
+      out_cmat.push_back(a);
+    }
+  }
```

Comment:
Can we add a method to `DiscretizationVisualizationWriterMesh` that appends field data? Similar to `append_result_data_vector_with_context`? Then we don't have to expose the underlying visualisation manager of the `DiscretizationVisualizationWriterMesh` (see my previous comment) and the implementation here should become cleaner.

Reply:
see comment above.